### ISSUE TITLE
Helm Chart broken

### ISSUE THREAD

**Issue by xorl** [kubetail-org/helm-charts#36]
Followed the instructions, yet the helm chart throws an error. 

`Error: INSTALLATION FAILED: template: kubetail/templates/deployment.yaml:33:28: executing "kubetail/templates/deployment.yaml" at <include "kubetail.config" .>: error calling include: template: kubetail/templates/_helpers.tpl:110:10: executing "kubetail.config" at <.>: wrong type for value; expected chartutil.Values; got string`

Do KUBETAIL_CSRF_SECRET and KUBETAIL_SESSION_SECRET need to be set before attempting a deployment?

**Comment by amorey** [comment]
Hmm… are you using a custom value for kubetail.config? If you have a custom values.yaml file, it would help to see it so I can debug.

If the secret values are null, they will be autogenerated, stored in a Secret, then reused on subsequent deployments.

**Comment by amorey** [comment]
I just tried installing again from scratch using the latest helm chart and I didn't see this error. Let me know more details about your setup and I'll dig into this further.

**Comment by xorl** [comment]
I just added it and attempted to deploy from the instructions provided, no custom values file etc. copy and pasted exactly from command line.

```
 ➜ helm install kubetail kubetail/kubetail --namespace kubetail --create-namespace
Error: INSTALLATION FAILED: template: kubetail/templates/deployment.yaml:33:28: executing "kubetail/templates/deployment.yaml" at <include "kubetail.config" .>: error calling include: template: kubetail/templates/_helpers.tpl:110:10: executing "kubetail.config" at <.>: wrong type for value; expected chartutil.Values; got string
➜  
```

**Comment by xorl** [comment]
Figured it out, line 110 _helpers.tpl

```
{{- tpl . . | nindent 0 }}
```
should be:
```
{{- tpl . $ | nindent 0 }}
```

Works without error once this change is applied.
helm version.BuildInfo{Version:"v3.13.3", GitCommit:"c8b948945e52abba22ff885446a1486cb5fd3474", GitTreeState:"clean", GoVersion:"go1.20.11"}

**Comment by amorey** [comment]
Thanks for the bugfix! It would have taken me a while to figure that out. I fixed it and pushed a new version (v0.4.3).

Here's my helm version:
```
version.BuildInfo{Version:"v3.14.0", GitCommit:"3fc9f4b2638e76f26739cd77c7017139be81d0ea", GitTreeState:"clean", GoVersion:"go1.21.5"}
```

I also tried with v3.14.4 and didn't see the error. Any idea why we're seeing different behavior? Have you modified your helm installation at all?
